pom.xml

```diff
@@ -165,6 +165,8 @@
 	<properties>
 		<!-- Build Properties -->
 		<java.version>17</java.version>
+		<maven.compiler.release>${java.version}</maven.compiler.release>
```

Comment:
This is needed due to https://github.com/apache/maven-apache-parent/commit/b938653652022a5754d7398168d3f897e55228af